modules/ppcp-button/src/Endpoint/ApproveOrderEndpoint.php

```diff
@@ -145,10 +145,8 @@ public function handle_request(): bool {
 
 			$order = $this->api_endpoint->order( $data['order_id'] );
 
-			$payment_source = $order->payment_source();
-			assert( $payment_source instanceof PaymentSource );
-
-			if ( $payment_source->name() === 'card' ) {
+			$payment_source = $order->payment_source() ?? null;
```

Comment:
is `?? null` needed? I think it already returns `PaymentSource|null`. And afaik `??` checks only for null, so it's like "assign null if it is null".

Reply:
yeah, no need for checking `null` here, thanks!